plugins/lsp/language-servers/nil-ls-settings.nix

```
@@ -0,0 +1,65 @@
+{ lib, helpers }:
+# Options:
+#  - https://github.com/oxalica/nil/blob/main/docs/configuration.md?plain=1#
+with lib;
+{
+  formatting = {
+    command = helpers.defaultNullOpts.mkListOf types.str "null" ''
+      External formatter command (with arguments).
+      It should accepts file content in stdin and print the formatted code into stdout.
+    '';
+  };
+
+  diagnostics = {
+    ignored = helpers.defaultNullOpts.mkListOf types.str [ ] ''
```

```suggestion
    ignored = helpers.defaultNullOpts.mkListOf types.str "[]" ''
```
Otherwise, this would translate in an empty default in the documentation as `toString [ ]` is `""`.

---

Ah, makes sense. Forgot that was for documentation.